inc/commonitilobject.class.php

```diff
@@ -7561,6 +7561,9 @@ function getITILTemplateToUse(
       $itilcategories_id = 0,
       $entities_id = -1
    ) {
+      if (!$type && $this->getType() != Ticket::getType()) {
```

Comment:
I do not understand why `$this->getType() != Ticket::getType()` as we previously check for `if ($this->getType() == Ticket::getType()) {`.

Reply:
Got it.